packages/ledgerstate/color.go

```diff
@@ -0,0 +1,228 @@
+package ledgerstate
+
+import (
+	"bytes"
+	"sort"
+
+	"github.com/iotaledger/hive.go/datastructure/orderedmap"
+	"github.com/iotaledger/hive.go/marshalutil"
+	"github.com/iotaledger/hive.go/stringify"
+	"github.com/mr-tron/base58"
+	"golang.org/x/xerrors"
+)
+
+// region Color ////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+// ColorLength represents the length of a Color (amount of bytes).
+const ColorLength = 32
+
+// Color represents a marker that is associated to a token balance and that can give tokens a certain "meaning".
+type Color [ColorLength]byte
+
+// ColorFromBytes unmarshals a Color from a sequence of bytes.
+func ColorFromBytes(bytes []byte) (color Color, consumedBytes int, err error) {
+	marshalUtil := marshalutil.New(bytes)
```

Comment:
`bytes` collides with the import `"bytes"`. Not that important but could lead to some issues/confusion.